docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -38,15 +38,19 @@ You can set up CSPM for AWS either by enrolling a single cloud account, or by en
 . Click *Add Cloud Security Posture Management (CSPM)*.
 . Select *AWS*, then either *AWS Organization* to onboard multiple accounts, or *Single Account* to onboard an individual account.
 . Give your integration a name that matches the purpose or team of the AWS account/organization you want to monitor, for example, `dev-aws-account`.
+. (Optional) click **Advanced settings** to deploy the integration using agentless technology (beta). 
```

Comment:
Fixes typo, and uses a formal [beta admonition](https://github.com/elastic/docs/blob/master/README.asciidoc#using-the-beta-admonition):
```suggestion
. beta:[] (Optional) Click **Advanced settings** to deploy the integration using agentless technology. 
```